images/virt-artifact/patches/README.md

```diff
@@ -62,10 +62,26 @@ Do not create Kubevirt APIService.
 #### `015-rename-core-resources.patch`
 Replace "kubevirt" with "kubevirt-internal-virtualziation" in the core resource names.
 
+#### `016-rename-install-strategy-labels.patch`
```

Comment:
Need to increase index

Reply:
It'll be fixed with removing `016-rename-apigroups-in-starred-rbac.patch` in #218.